valhalla/thor/map_matcher.h

```diff
@@ -28,7 +28,7 @@ class MapMatcher {
            const std::vector<meili::EdgeSegment>& edge_segments,
            const std::shared_ptr<sif::DynamicCost>* mode_costing,
            const sif::TravelMode mode,
-           std::vector<std::pair<baldr::GraphId, baldr::GraphId>>& disconnected_edges,
+           std::unordered_map<baldr::GraphId, baldr::GraphId>& disconnected_edges,
```

Comment:
this is dubious for the reason i mentioned before. once its unordered you cant know what part of the rout eyou are talking about if the route has loops